README.md

```diff
@@ -48,6 +48,6 @@ npm install
 npm install -g grunt-cli
 ```
 
-Then to run the tests use ```grunt fh:test```
+Then to run the tests use ```npm test```
```

Comment:
Using "npm test" is a little more standard and means people don't need to have grunt globally installed.